src/couch_pse_tests/src/cpse_test_attachments.erl

```diff
@@ -12,6 +12,7 @@
 
 -module(cpse_test_attachments).
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
All functions in this module need to be exported. Since we have a test discovery based on `cpse_` prefix (see https://github.com/apache/couchdb/blob/master/src/couch_pse_tests/src/cpse_gather.erl#L31). Which means that it is ok to use `export_all` here.